gio/Gir.toml

```diff
@@ -799,6 +797,13 @@ status = "generate"
     # Better API with `IntoIter`
     ignore = true
 
+[[object]]
+name = "Gio.FileCopyFlags"
+status = "generate"
+    [[object.member]]
+    name = "target_default_modified_time"
+    version = "2.80"
```

Comment:
Information missing from the gir files? Do we have PRs upstream for fixing it?

Reply:
Sadly, for enums/flags, the situation is a bit difficult. The since annotation is only properly parsed if it is in a new line, requiring documenting every member separately which is not done across projects. Hopefully something that will get fixed at some point

Reply:
Oh I see. Well, let's hope we'll do it not too far in the future so things like this won't be needed anymore.